data/net/minecraft/world/effect/MobEffectInstance.mapping

```diff
@@ -3,8 +3,44 @@ CLASS net/minecraft/world/effect/MobEffectInstance
 		COMMENT A hidden effect which is not shown to the player.
 	FIELD noCounter Z
 		COMMENT True if potion effect duration is at maximum, false otherwise.
+	METHOD <init> (Lnet/minecraft/world/effect/MobEffect;)V
+		ARG 1 effect
+	METHOD <init> (Lnet/minecraft/world/effect/MobEffect;I)V
+		ARG 1 effect
+		ARG 2 duration
+	METHOD <init> (Lnet/minecraft/world/effect/MobEffect;II)V
+		ARG 1 effect
+		ARG 2 duration
+		ARG 3 amplifier
+	METHOD <init> (Lnet/minecraft/world/effect/MobEffect;IIZZ)V
+		ARG 1 effect
+		ARG 2 duration
+		ARG 3 amplifier
+		ARG 4 ambient
+		ARG 5 visible
+	METHOD <init> (Lnet/minecraft/world/effect/MobEffect;IIZZZ)V
+		ARG 1 effect
+		ARG 2 duration
+		ARG 3 amplifier
+		ARG 4 ambient
+		ARG 5 visible
+		ARG 6 showIcon
+	METHOD <init> (Lnet/minecraft/world/effect/MobEffect;IIZZZLnet/minecraft/world/effect/MobEffectInstance;)V
+		ARG 1 effect
+		ARG 2 duration
+		ARG 3 amplifier
+		ARG 4 ambient
+		ARG 5 visible
+		ARG 6 showIcon
+		ARG 7 hiddenEffect
+	METHOD <init> (Lnet/minecraft/world/effect/MobEffectInstance;)V
+		ARG 1 effectInstance
 	METHOD applyEffect (Lnet/minecraft/world/entity/LivingEntity;)V
 		ARG 1 entity
+	METHOD compareTo (Ljava/lang/Object;)I
+		ARG 1 effectInstance
```

Comment:
This is mapping the synthetic method -- the proper method that should be mapped here has the parameter of `MobEffectInstance`. For now, please remove this mapping (either manually or using the `sanitizeData` task).